_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -0,0 +1,319 @@
+---
+id: 88407d
+name: Inline link has distinguishable style not based on color alone or content that indicates it is a link
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style not based on color alone or have content that indicates they are links.
+accessibility_requirements:
```

Comment:
Do atomic rules not include any accessibility requirements?

Reply:
This (partially) covers SC 1.4.1. But it would not support the fail-on-fail relationship that rules need to have. Therefore the accessibility requirements end up only in the composite rule.